### PR TITLE
Added toast message rather than loading component

### PR DESCRIPTION
**Pull request by fish-sushi**

### CODE REVIEW

File: app/javascripts/app.jsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import EnvChecker from './helpers/envChecker';
 // redux stuff
 import { store, getHistoryObject } from './store';
 import Root from './root';
+// import 'notie/dist/notie.min.css';
```
Comment (parkkoon) [inline app/javascripts/app.jsx]: Please delete this comment.

File: app/javascripts/components/makeGame/actions.jsx [hunk 2]
```diff
@@ -15,16 +16,26 @@ export const ACTION_TYPES = {
...
+        text: 'Finish to making a slot machine',
+      });
       dispatch({
         type: ACTION_TYPES.SUCCEED_TO_MAKE_GAME,
         payload: transaction,
       });
     } catch (err) {
+      Toast.notie.alert({
+        type: 'error',
+        text: 'There wat an error for making a slot machine',
```
Comment (parkkoon) [inline app/javascripts/components/makeGame/actions.jsx]: I think it is a typing error instead of 'was'

**Review by parkkoon — CHANGES_REQUESTED** [review]

**Review by c0smic0wl — APPROVED** [review]

File: app/javascripts/app.jsx (hunk 1 above)
Comment (fish-sushi) [reply app/javascripts/app.jsx]: Thx

File: app/javascripts/components/makeGame/actions.jsx (hunk 2 above)
Comment (fish-sushi) [reply app/javascripts/components/makeGame/actions.jsx]: Thx~

**Pull request merged**